ansible/container-projects.yaml

```diff
@@ -26,19 +26,16 @@
          links:
            - projects-mysql:mysql
          volumes:
-           - /data/files/projects/:/home/redmine/data/
+           - /data/files/projects/configuration.yml:/usr/src/redmine/config/configuration.yml:ro
+           - /data/files/projects/files:/usr/src/redmine/files
+           - /data/files/projects/plugins:/usr/src/redmine/plugins:ro
+           - /data/files/projects/themes:/usr/src/redmine/themes:ro
          env:
-           DB_HOST="mysql"
-           DB_USER="projects"
-           DB_PASS="{{ lookup('hashi_vault', 'secret=secret/projects/mysql/password:value') }}"
-           DB_NAME="projects"
-           DB_TYPE="mysql"
-           REDMINE_HTTPS="true"
-           REDMINE_ATTACHMENTS_DIR="/home/redmine/data/files"
-           NGINX_MAX_UPLOAD_SIZE="1024m"
-           SMTP_HOST="smtprelay.piratenpartei.ch"
-           SMTP_PORT="25"
-           SMTP_ENABLED="true"
+           REDMINE_DB_DATABASE="projects"
+           REDMINE_DB_MYSQL="mysql"
+           REDMINE_DB_PASSWORD="{{ lookup('hashi_vault', 'secret=secret/projects/mysql/password:value') }}"
+           REDMINE_DB_USERNAME="redmine"
```

Comment:
Why do you want to switch the username from "projects" to "redmine"?

Reply:
Fixed